gateway/src/shard/tls.rs

```diff
@@ -60,10 +60,7 @@ impl TlsError {
 
 impl Display for TlsError {
     // If TlsErrorType is an empty type f is not used.
-    #[cfg_attr(
-        not(any(feature = "native", feature = "rustls-native-roots")),
-        allow(unused)
-    )]
+    #[cfg_attr(feature = "rustls-webpki-roots", allow(unused))]
```

Comment:
Does this not warn with unused if no feature is selected?

Reply:
No because this entire module is `cfg` gated